Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyViewContextMenuPlugin.cxx

```diff
@@ -73,12 +77,17 @@ class qSlicerSubjectHierarchyViewContextMenuPluginPrivate: public QObject
   QAction* InteractionModePlaceAction = nullptr;
 
   QAction* MaximizeViewAction = nullptr;
+  QAction* FitSliceViewAction = nullptr;
+  QAction* CenterThreeDViewAction = nullptr;
   QAction* CopyImageAction = nullptr;
   QAction* ConfigureSliceViewAnnotationsAction = nullptr;
+  QAction* ToggleTiltLockAction = nullptr;
+
 
   vtkWeakPointer<vtkMRMLInteractionNode> InteractionNode;
   vtkWeakPointer<vtkMRMLAbstractViewNode> ViewNode;
   vtkWeakPointer<vtkMRMLLayoutNode> LayoutNode;
+  vtkWeakPointer<vtkMRMLCameraWidget> CameraWidgetPtr;
```

Comment:
Rename to CameraWidget (since we almost always refer to all VTK and Qt objects by their pointer, using the `Ptr` suffix would be redundant).

Reply:
Renamed .